frontend/src/pages/Profile.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { user } from '../reducers/user'
+import { logout } from '../reducers/user'
```

Comment:
Here you could also get the `getSecretMessage`
`import { user, logout, getSecretMessage } from '../reducers/user'`

and then dispatch it further down:
```
dispatch(getSecretMessage())
const secretMessage = useSelector((store) => store.user.login.secretMessage)
```